modules/nw-ne-ways-to-manage-ingress-traffic.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-ways-to-manage-ingress-traffic_{context}"]
+= Ways to manage ingress controller traffic
+The following table provides an overview of ways that Administrators can manage their Ingress traffic:
+|===
```

Comment:
Suggest specifying column widths so that the second column isn't wrapping so much. Something like:

[cols="1,2",options="header"]